smibhid/lib/hid.py

```diff
@@ -1,9 +1,10 @@
 from ulogging import uLogger
 from asyncio import get_event_loop
-from slack_api import Wrapper
```

Comment:
Not sure about this kind of import. I'd rather it be explicitly relative or absolute. Will approve but need to discuss it tomorrow.

Reply:
These are all relative imports? But I know I've mixed relative and partial absolutes elsewhere due to weirdness in the pico failing to resolve them. We can test some more tomorrow and work out how to resolve them if possible.